### PR TITLE
Updating linux pipeline from BASE folder

### PR DESCRIPTION
**Pull request by eddynaka**
Fix Issue # .

## Changes
(Please provide a brief description of the changes here.)

### Checklist
- [ ] I ran Unit Tests locally.
- [ ] CHANGELOG.md updated with one line description of the fix, and a link to the original issue if available.

For significant contributions please make sure you have completed the following items:

- [ ] Design discussion issue #
- [ ] Changes in public surface reviewed

The PR will trigger build, unit tests, and functional tests automatically. Please follow [these](https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/.github/CONTRIBUTING.md) instructions to build and test locally.

### Notes for authors:
- FxCop and other analyzers will fail the build. To see these errors yourself, compile localy using the Release configuration.

### Notes for reviewers:

- We support [comment build triggers](https://docs.microsoft.com/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#comment-triggers)
  - `/AzurePipelines run` will queue all builds
  - `/AzurePipelines run <pipeline-name>` will queue a specific build

### CODE REVIEW

File: BASE/.vsts/linux-build.yml [hunk 1]
```diff
@@ -1,47 +1,50 @@
 pool:
   vmImage: 'ubuntu-16.04'
```
Comment (cijothomas) [inline BASE/.vsts/linux-build.yml:2]: lets move to newer version. 16 is getting older.. 18.04.5

Comment (cijothomas) [reply BASE/.vsts/linux-build.yml:2]: https://wiki.ubuntu.com/Releases

**Review by cijothomas — APPROVED** [review]

File: BASE/.vsts/linux-build.yml (hunk 1 above)
Comment (eddynaka) [reply BASE/.vsts/linux-build.yml:2]: i tried...using "ubuntu-latest" but two tests started to fail in MetricsTest

Comment (cijothomas) [reply BASE/.vsts/linux-build.yml:2]: :D lets pick 18.04.5 and try. If it fails, then we have an issue with latest linux.

Comment (eddynaka) [reply BASE/.vsts/linux-build.yml:2]: today, ubuntu-latest = 18.04

**Review by cijothomas — APPROVED** [review]

**Pull request merged**